src/main.ts

```diff
@@ -56,20 +57,19 @@ export async function run() {
       return Promise.reject(new Error('Not Supported Flow'))
     }
 
-    if (inputs.POLARIS_SERVER_URL) {
+    if (validatePolarisInputs()) {
       const polarisCommandFormatter = new SynopsysToolsParameter(tempDir)
       const polarisAssessmentTypes: Array<string> = JSON.parse(inputs.POLARIS_ASSESSMENT_TYPES)
       formattedCommand = formattedCommand.concat(polarisCommandFormatter.getFormattedCommandForPolaris(inputs.POLARIS_ACCESS_TOKEN, inputs.POLARIS_APPLICATION_NAME, inputs.POLARIS_PROJECT_NAME, inputs.POLARIS_SERVER_URL, polarisAssessmentTypes))
```

Comment:
can we just pass inputs in getFormattedCommandParameters n extract input fields inside that